docs/dev/libvirt-howto.md

```diff
@@ -151,6 +151,15 @@ $ sudo firewall-cmd --get-active-zones
 FedoraWorkstation
   interfaces: enp0s25 tun0
 ```
+
+*NOTE:* The installer creates a libvirt network and a `tt0` bridge. It is
```

Comment:
Point out that this bridge name is [configurable][1]?

[1]: https://github.com/openshift/installer/blob/v0.11.0/pkg/types/libvirt/network.go#L7

Reply:
I didn't know about it. Force pushing a new change with instructions on how to configure it in the install-config.yaml file, thanks!!